src/test/java/com/wikia/webdriver/testcases/mercurytests/SEOTests.java

```diff
@@ -44,7 +44,7 @@ public void MercurySEOTest_001_MetaTags_CanonicalLink() {
     SEOUtils seo = new SEOUtils(driver);
     leftNav.openMercuryArticleByName(wikiURL, MUPPET_MAIN_PAGE);
 
-    // Uncomment after issue is fixed - related to HG-668
+    //TODO: Uncomment after issue is fixed - related to HG-668
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)